doc/api.md

```diff
@@ -203,25 +203,25 @@ Note that there is a 1:1 correspondence between payload fields and DB fields (bu
+| ------------------------- | -------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------- |
+| `device_id`               | `id`                 | Device ID that will be used to identify the device.                                                                                                                                                                                                                  | UO834IO                                       |
+| `service`                 | `service`            | Name of the service the device belongs to (will be used in the fiware-service header).                                                                                                                                                                               | smartGondor                                   |
+| `service_path`            | `subservice`         | Name of the subservice the device belongs to (used in the fiware-servicepath header).                                                                                                                                                                                | /gardens                                      |
+| `entity_name`             | `name`               | Name of the entity representing the device in the Context Broker                                                                                                                                                                                                     | ParkLamplight12                               |
+| `entity_type`             | `type`               | Type of the entity in the Context Broker                                                                                                                                                                                                                             | Lamplights                                    |
+| `timezone`                | `timezone`           | Time zone of the sensor if it has any                                                                                                                                                                                                                                | America/Santiago                              |
+| `timestamp`               | `timestamp`          | Optional flag about whether or not to addthe `TimeInstant` attribute to the device entity created, as well as a `TimeInstant` metadata to each attribute, with the current timestamp. With NGSI-LD, the Standard `observedAt` metadata attribute is created instead. | true                                          |
```

Comment:
```suggestion
| `timestamp`               | `timestamp`          | Optional flag about whether or not to addthe `TimeInstant` attribute to the device entity created, as well as a `TimeInstant` metadata to each attribute, with the current timestamp. With NGSI-LD, the Standard `observedAt` property-of-a-property is created instead. | true                                          |
```

As far as I understand, in NGSI-LD the concept of metadata doesn't exist natively and the right term is property-of-a-preoperty (although I may be wrong).